JRC Publications Repository.js

```diff
@@ -67,10 +67,21 @@ function getSearchResults(doc, checkOnly) {
 function detectWeb(doc, url) {
 	var type = ZU.xpath(doc, '//meta[@name="DC.type"]/@content');
 	if (type.length>0) {
-		if (mappingTable[type[0].value]) {
-			return mappingTable[type[0].value];
-		} else {//generic fallback
-			Z.debug('Unrecognized item type: ' + type[0].value);
+		if (mappingTable[type[0].textContent]) {
+			return mappingTable[type[0].textContent];
```

Comment:
Okay, let me just understand this a little more: Whenever I have an xpath which ends with `@attribute`, then I should use `.value`. And for nodes (different form of xpath) we use `.textContent`. Is that correct?